src/sass/variables/_modals.scss

```diff
@@ -1 +1 @@
-$uds-modal-overlay-background-color: rgba(25, 25, 25, .75);
+$uds-modal-overlay-background-color: rgb(25, 25, 25, .75);
```

Comment:
I'm getting an error message in the compiler that this has to be **rgba** since it has 4 values not 3: 
 formatted: "Error: wrong number of arguments (4 for 3) for `rgb'\n" +
    '        on line 1 of src/sass/variables/_modals.scss\n' +
    '        from line 48 of src/sass/bootstrap-asu.scss\n' +
    '        from line 4 of sass/theme.scss\n' +
    '>> $uds-modal-overlay-background-color: rgb(25, 25, 25, .75);\n' +
    '\n' +